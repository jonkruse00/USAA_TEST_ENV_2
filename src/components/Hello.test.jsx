import React from 'react';
import renderer from 'react-test-renderer';
import Hello from './Hello';

describe('Hello', () => {
  it('should render Hello component', () => {
    const tree = renderer.create(<Hello />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});