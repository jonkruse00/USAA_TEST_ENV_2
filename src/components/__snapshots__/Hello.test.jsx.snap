// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello should render Hello component 1`] = `
<h1>
  Hello world!
</h1>
`;
